File: 002-Backend/LaPlata.API/LaPlata.Domain/DTOs/Despesas/ContaVariavel/ReadContaVariavelDTO.cs
namespace LaPlata.Domain.DTOs
{
    public class ReadContaVariavelDTO
    {
        public int Id { get; set; }
        public string Descricao { get; set; }
        public int DiaVencimento { get; set; }
        public ReadCategoriaDTO Categoria { get; set; }
        public List<ReadPagamentoContaVariavelDTO> Pagamentos { get; set; }
    }
}

File: 003-Frontend/HomeTools.Front/src/app/interfaces/api-dto/Financas/readContaVariavelDto.ts
import { Categoria } from "../../categoria";
import { PagamentoContaVariavelDto } from "./readPagamentoContaVariavelDto";

export interface ReadContaVariavelDto {
  Id: number;
  Descricao: string;
  DiaVencimento: number;
  Categoria: Categoria;
  Pagamentos: PagamentoContaVariavelDto[];
}
